### DISCUSSION TITLE
Status of Ray Tracing & Mesh Shaders

### DISCUSSION THREAD

**Discussion by Cysmic** [KhronosGroup/MoltenVK#2079] [General]
Was wondering what the timeline looks like for implementing ray tracing and mesh shaders for moltenvk. Is it something being worked on right now?

**Comment by billhollings** [comment]
Ray tracing is a large project, and we are currently talking with customers about funding its development.

Some initial work has been made on Acceleration Structures in a [WIP PR](https://github.com/KhronosGroup/MoltenVK/pull/1967).
